common/utils.js

```diff
@@ -57,12 +57,16 @@ function printLog(log) {
     console.log(JSON.stringify({ log }, null, 2));
 }
 
+const isString = (arg) => {
+    return typeof arg === 'string';
+};
+
 const isNonEmptyString = (arg) => {
-    return typeof arg === 'string' && arg !== '';
+    return isString(arg) && arg !== '';
 };
 
-const isString = (arg) => {
-    return typeof arg === 'string';
+const isStringLowercase = (arg) => {
+    return isString(arg) && arg === arg.toLowerCase();
```

Comment:
```suggestion
    return isNonEmptyString(arg) && arg === arg.toLowerCase();
```
should check for non-empty by default